reverse-proxy.md

```diff
@@ -271,6 +271,7 @@ server {
         proxy_set_header Host $host;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_read_timeout 3600; # set it to the same value as this env: https://github.com/nextcloud/all-in-one#how-to-adjust-the-max-execution-time-for-nextcloud
```

Comment:
One day:
```suggestion
        proxy_read_timeout 86400s;
```

Reply:
```suggestion
        proxy_read_timeout 86400s; # This is equal to 24h. If you have so big files that they need more than 1 day to upload, it will fail tu upload them and lead to a 504 error
```

Reply:
The upload won't fail as already said in #979 it will just show an error, but the file will be uploaded without error

Reply:
ah I see. So then just use your suggestion. Still it is an error that gets triggered when exeeding this limit?

Reply:
> ah I see. So then just use your suggestion. Still it is an error that gets triggered when exeeding this limit?

The error will be shown when exceeding one day, but even if the file takes 3 days to upload, it will show the error after 3 days, but the file be saved on the cloud without errors

Reply:
okay, then just use your suggested value 👍

Reply:
change seems good to me, fwiw